### DISCUSSION TITLE
cannot log in to github.com  using my user and password

### DISCUSSION THREAD

**Discussion by maxcli** [community/community#45579] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Screen shows:
We just sent your authentication code via email to xxxx@hotmail.com. The code will expire at 1:28PM PST

I never get the  auth code
I checked my junk folder
And I tried to resend  . 
Is github auth broken, or am I missing something?
Thanks, Peter

**Comment by jge162** [comment]
maybe the message is delayed or being blocked by hotmail. All services are up for GitHub. 

https://www.githubstatus.com/
<img width="1429" alt="Screen Shot 2023-01-27 at 12 41 41 PM" src="https://user-images.githubusercontent.com/31228460/215192997-ee49c461-26ba-48ae-9ad0-4f4ea6292ed3.png">

**Comment by maxcli** [comment]
I think Hotmail is working, but git is messed up...

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Reply by jge162** [reply]
Ok.

**Comment by maxcli** [comment]
Can anyone else help? I am locked out of the github account. is it possible the auth email address was changed to go somewhere else? I am 99% sure my user and pwd are ok. Thanks in advance.

**Reply by jge162** [reply]
You need to submit a support request. Have you done so yet?
